scripts/sync-workflow.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
```

Comment:
I'm not sure what the best name for this file would be: I clearly didn't give it much thought. Suggestions welcome!